src/stuf/stuf_bg/constants.py

```diff
@@ -17,6 +24,37 @@
 STUF_BG_EXPIRY_MINUTES = 5
 
 
+type DateOfBirthPrecisionType = Literal["date", "year_month", "year"]
+
+
+class NaturalPersonDetails(BaseModel):
```

Comment:
this is not a constant, it's a data container. Probably better to put this in a `data.py`. (I'd normally go for `models.py`, but that's too confusing with django models)